src/main/java/eu/dissco/backend/service/SpecimenService.java

```diff
@@ -261,11 +172,11 @@ private JsonApiListResponseWrapper wrapListResponse(
   }
 
   private JsonApiListResponseWrapper wrapListResponseSearchResults(
-      Pair<Long, List<DigitalSpecimenWrapper>> digitalSpecimenSearchResult,
+      Pair<Long, List<eu.dissco.backend.schema.DigitalSpecimen>> digitalSpecimenSearchResult,
```

Comment:
You have a few places where you reference the full path of DigitalSpecimen, but the object is already imported. Would be cleaner if you removed the path

Reply:
Yes fixed